src/core/webidl.js

```diff
@@ -664,6 +664,11 @@ function findDfn(parent, name, definitionMap, type, idlElem) {
     // an explicitly empty [for], try <dfn> that inherited a [for].
     if (dfns.length === 0 && parent === "" && dfnForArray.length === 1) {
       dfns = dfnForArray;
+    } else if (type === "interface" && dfnForArray.length) {
```

Comment:
Why not `interface mixin`, `dictionary`, `namespace`, `typedef`, etc. but only interface?

Reply:
This is why we pay you the big bucks, @saschanaz! :) yes, it should check any top level container type.